lib/form/html.js

```diff
@@ -189,7 +189,7 @@ const renderer = (type, errors) => {
 
           // Adds delete button if instances are greater than minimum
           if (input.inputs.length > input.repeatable.min) {
-            render += `<button type="button" class="delete--button" id="${input.id}--delete--${index}">Delete</button>`;
+            render += `<button type="button" class="form--delete" id="${input.id}--delete--${index}">Delete</button>`;
```

Comment:
Well, that's why this line is changed. It should be `form--delete`. The link you have to `HTML generation in Content Types` is the link to this line...so, that (this) line is, ya know, already updated.

No other references to `delete-button` exist that I know of.